### DISCUSSION TITLE
Elefantes

### DISCUSSION THREAD

**Discussion by vish-blog** [vish-blog/comments#316] [Imagens]
Post de referência: https://www.vish.blog.br/imagens/tirinhas/elefantes/

**Comment by brunoaugusto** [comment]
Malditos ninjas cortadores de cebola. Mal posso ver seus movimentos

**Comment by one-eyed-willy** [comment]
Foi uma excelente e bastante emocionante *webcomic* (principalmente o primeiro diálogo com a "Dona Morte"). É que... não me entendam mal, mas o quão errado teria dado a minha vida pra eu precisar andar num elefante?

**Comment by najarin-the-loremaster** [comment]
> [Willy Caolho](</perfil/?user=one-eyed-willy>) disse [terça-feira, 17 de maio de 2022](<https://www.vish.blog.br/imagens/tirinhas/elefantes/#comment_1129027196>):

> Foi uma excelente e bastante emocionante *webcomic* (principalmente o primeiro diálogo com a "Dona Morte"). É que... não me entendam mal, mas o quão errado teria dado a minha vida pra eu precisar andar num elefante?

A frase ficou um pouco clichê e até meio piegas, admito. Mas eu apenas traduzi o que estava no último quadro da imagem original :wink:
